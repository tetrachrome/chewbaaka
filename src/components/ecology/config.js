/**
 * ecology/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 19, 2020
 * Updated  : Jul 20, 2020
 */

export const config = {
  pageProps: {
    coverImage: "",
    title: "Ecology",
    subtitle: "Subtitle TBD ...",
    pageMenuItems: [
      {
        title: "Ecosystem and Habitat",
        tocImageFilename: ""
      },
      {
        title: "Eco-management",
        tocImageFilename: ""
      },
      {
        title: "Research",
        tocImageFilename: ""
      },
    ],
  },
  contentPageIntro: {
    title: "Ecology",
    content: "TBD ..."
  },
  contentPageSections: {
    section_EcosystemAndHabitat: {
      intro: {
        title: "Ecosystem and Habitat",
        content: "Understanding the relationships and interactions of elements in the ecosystem and habitat " +
                "where cheetahs live is crucial for any conservation and management efforts. " +
                "Understanding of subjects in this area can help answer questions such as " +
                "“what animals do cheetahs prey on and eat” and “what are its roles in its habitat and " +
                "relationships with other species of animals”."
      },
      subsections: {
        subsection_WhereCheetahsLive: {
          title: "Where Cheetahs Live",
          contents: {
          }
        },
        subsection_TheCheetahsPrey: {
          title: "The Cheetah's Prey",
          contents: {
          }
        }
      }
    },
    section_Ecomanagement: {
      intro: {
        title: "Eco-management",
        content: "TBD ..."
      },
      subsections: {
      }
    },
    section_Research: {
      intro: {
        title: "Research",
        content: "TBD ..."
      },
      subsections: {
      }
    }
  }
};
