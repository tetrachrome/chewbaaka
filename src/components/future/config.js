/**
 * future/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 14, 2020
 * Updated  : Jul 22, 2020
 */

export const config = {
  pageProps: {
    coverImage: "",
    title: "Future",
    subtitle: "Subtitle TBD ...",
    pageMenuItems: [
      {
        title: "Conservation",
        tocImageFilename: ""
      },
      {
        title: "Sustainable Development",
        tocImageFilename: ""
      },
      {
        title: "Outreach and Education",
        tocImageFilename: ""
      }
    ],
  },
  contentPageSections: {
    section_Conservation: {
      intro: {
        title: "Conservation",
        content: "Content TBD ..."
      },
      subsections: {
        subsection_WhatIsConservation: {
          title: "What is Conservation?",
          contents: {
          }
        },
        subsection_CheetahAmbassadors: {
          title: "Cheetah Ambassadors",
          contents: {
            paragraph_Chewbaaka:
              "Hi there, my name is Chewbaaka. I was named after an animal " +
              "in a movie. I have lived at CCF since I was 3 weeks old. " +
              "I am there cheetah ambassador and I get to meet many people. " +
              "The staff at CCF tell everyone about me and all the wild " +
              "cheetah. I have a special place where I can run like the wind.",
            paragraph_Mekondyo:
              'Hello, my name is Mekondyo. Mekondyo means "struggle" in the ' +
              "Oshiwambo language. I was born on a farm north of Otjiwarango " +
              "but I now live on farmland west of Otjiwarango. It is very " +
              "beautiful. I can see the Waterberg Plateau far away. " +
              "I am 5 years old now and I will tell you my story as you " +
              "explore this museum."
          }
        },
      },
    },
    section_SustainableDevelopment: {
      intro: {
        title: "Sustainable Development",
        content: "Content TBD ..."
      },
      subsections: {
        subsection_MissionPossible: {
          title: "Mission Possible",
          contents: {
          }
        },
        subsection_FutureFarmersOfAfrica: {
          title: "Future Farmers of Africa",
          contents: {
          }
        },
        subsection_LivestockGuardingDogs: {
          title: "Livestock Guarding Dogs",
          contents: {
          }
        }
      }
    },
    section_OutreachAndEducation: {
      intro: {
        title: "Outreach and Education",
        content: "Content TBD ..."
      },
      subsections: {
        subsection_FieldResearch: {
          title: "Field Research",
          content: {
          }
        },
        subsection_InternshipsAndVolunteering: {
          title: "Internships and Volunteering",
          content: {
          }
        }
      }
    }
  },
};
