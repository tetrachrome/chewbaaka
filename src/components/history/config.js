/**
 * history/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 07, 2020
 * Updated  : Jul 10, 2020
 */

export const config = {
  pageProps: {
    coverImage: "../history/assets/cheetah-mother-and-cubs-on-termite-mound.jpg",
    title: "History",
    subtitle: "Learn about the evolution and migration of the species, its relationship with man throughout history, and its population and ranges.",
    pageMenuItems: [
      "Evolution",
      "Cheetah and Man",
      "Range and Population"
    ],
  },
  contentPageIntro: {
    title: "History of the Cheetah",
    content: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
  },
  contentPageSections: {
    section_Evolution: {
      intro: {
        title: "Evolution",
        content: "The oldest fossils place  cheetahs in North America in what is now Texas, Nevada and Wyoming. Cheetahs were common throughout Asia, Africa, Europe and North America until the end of the last Ice Age, about 10,000 years ago, when massive climatic changes caused large numbers of mammals to disappear."
      },
      subsections: {
        subsection_EvolutionOfCats: {
          title: "Evolution of Cats"
        }
      }
    },
    section_CheetahAndMan: {
      intro: {
        title: "Cheetah and Man",
        content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
      },
      subsections: {
        subsection_RelationshipsWithMan: {
          title: "Relationships with Man",
          contents: {
            paragraph_01: "Try resizing the window: You'll see that each sentence is on one line when the window is wide enough. Only when the window is too narrow for the whole sentence will the sentence be broken over several lines. When you remove the 'margin-right: -50%' and resize the window again, you'll see that the sentences will be broken already when the window is still twice as wide as the text lines. ",
            paragraph_Cheetah_And_Man_01:
              'Traditional African healers and witchdoctors used cheetah foot bones in spiritualistic rituals to symbolize fleet-footnedness and speed. ' +
              'They used bones from a wide variety of animals and techniques varied from tribe to tribe. Objects ' +
              'represented a person, a thing or mood fron the past, present or future, and are known as Divinatory sets.',
            paragraph_Cheetah_And_Man_02:
              'During the time of the Egyptian pharaohs, the cheetah was considered a goddess named ' +
              '"Mafdet". Pharaohs kept cheetahs was close companions, ' +
              "symbolic of Mafdet's protection of the royal throne.",
            paragraph_Cheetah_And_Man_03:
              'The San of Southern Africa ate cheetah meat as a symbol for speed, but it was not a main food in their diet.',
            paragraph_Cheetah_And_Man_04:
              'Kings wore cheetah skins for dignity. Trade in cheetah skin only started after European explorers began ' +
              'requesting them.',
          }
        },
        subsection_CheetahsInSports: {
          title: "Cheetahs In Sports",
          contents: {
            paragraph_The_Hunting_Leopard_01:
              'In c1700 BC, the Egyptians were the first to tame cheetahs. They admired the cheetah for its speed, ' +
              'hunting ability and beauty. They honored cheetahs as symbols of royalty and prestige.',

            paragraph_The_Hunting_Leopard_02:
              'The swiftest animal on earth became a cherished hunting companion of Pharaohs and royalty ' +
              'throughout Europe, Asia and India. Until the early 1900s, ownership of cheetahs was as important ' +
              'to these nobles as their love for gold. Cheetahs hunt by sight so they excelled in the sport ' +
              'known as "coursing". Hunts organized by royalty and nobles were for the challenge of sport, ' +
              'not for food. Hunts represented power and prestige.',

            paragraph_The_Hunting_Leopard_03:
              'By the 1500s the popularity of the cheetah as a hunting companion rivalled that of the dog. ' +
              'Cheetahs, the most easily tamed of the big cats, were caught, tamed and trained. Adults were ' +
              'used because cubs had not learned how to hunt. Tamed cheetahs formed a strong bond with their keepers.',

            paragraph_The_Hunting_Leopard_04:
              'Each cheetah rode to the hunt by horseback or on a cart. Its eyes were coved with a hood and uncovered ' +
              'when prey were sighted. The cat was released to chase down the prey then rewarded with meat fed from a ' +
              'wooden spoon.',

            paragraph_The_Hunting_Leopard_05:
              'Althrough cherished, pampered cheetahs were loved to near extinction and taken from the wild in great numbers. ' +
              'By the early 1900s, India and Iran were importing African cheetahs for the sport of "coursing", as their own ' +
              'wild population became too small.'
          }
        },
        subsection_CheetahsInArt: {
          title: "Cheetahs In Art",
          contents: {
            paragraph_Cheetah_In_Art_01:
              'Sumerians (Iraqi ancestors) were the first to use cheetahs in art.',

            paragraph_Cheetah_In_Art_02:
              'The cheetah and leopard have often been mistaken for each other. ' +
              'Cheetahs in early art were frequently called "panther" ' +
              'or "hunting leopard". Early artists called the true leopard "pardus" ' +
              'and described it as a cross between a cheetah (panther), and the lion (leo).',

            paragraph_Cheetah_In_Art_03:
              'Middle Age and Renaissance artists began drawing cheetahs and other ' +
              'animals more life-like. During this time of scientific exploration, ' +
              'detailed descriptions of animals provided incentive for reality in art. ' +
              'Artists included cheetahs more often in their paintings as the trade in ' +
              'live animals increased.',

            paragraph_Cheetah_In_Art_04:
              'Artists always pictured cheetahs as animals of speed and royalty.'
          }
        }
      }
    },
    section_PopulationAndRange: {
      intro: {
        title: "Range and Population",
        content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
      },
      subsections: {
        subsection_Namibia: {
          title: "Namibia - Cheetah Capital of the World",
          contents: {
            paragraph_01:
              "Namibia has the world's largest cheetah population. " +
              "Approximately 3000 cheetahs share the land with humans, " +
              "livestock, and wildlife. " +

              "Today, the status of the Namibian cheetah is stabilizing. " +
              "During the 1980s, the population of Namibina cheetahs " +
              "declined by half. In the 10-year period, nearly " +
              "7,000 cheetahs were removed from the wild. " +

              "In most countries where cheetahs live, their numbers " +
              "have been reduced to critical levels. The cheetah's " +
              "survival worldwide is in human hands.",

            paragraph_Namibia_Home_For_The_Future:
              "The greatest hope for the cheetah's survival lies in the " +
              'pristine countryside of Namibia. Nearly 1000 Namibian farmers ' +
              "control the fate of the country's cheetahs and the land they live.",
          }
        },
        subsection_RoadToExtinction: {
          title: "The Road to Extinction ?",
          contents: {
            paragraph_The_Road_To_Extinction_01:
              "World development, industrialization, automobiles, and aeroplanes...",

            paragraph_The_Road_To_Extinction_02:
              "Through the 1900s, man's inventions seemed limitless, yet there is " +
              "nothing man-made that rivals the speed and efficiency of the cheetah.",

            paragraph_The_Road_To_Extinction_03:
              "Scientists classified the cheetah as Acinonyx jubatus, yet nobles still " +
              'referred to it as the "hunting leopard". By the end of the 1800s, cheetahs' +
              'were a rarity in Asia Minor and Arabia because of their use in the sport ' +
              'of "coursing".',

            paragraph_The_Road_To_Extinction_04:
              'Although it appeared that the cheeatahs had a large range, their numbers ' +
              'within that range existed in small pockets. Cheetahs, farmers and their livestock ' +
              'all preferred open grasslands for their habitat. Increasing agricultural development ' +
              'and new settlements played havoc with remaining cheetah populations.',

            paragraph_Where_Did_The_Cheetah_Go_01:
              'As human population grew, the amount of land devoted to livestock farming ' +
              'steadily increased. Livestock filled the open land where cheetahs roamed. ' +
              'Natural prey became scarce. Farmers kill other large predators. Although ' +
              'game reserves protected them, cheetahs could not compete against hyoenas ' +
              'and lions. Farmlands offered cheetahs a safe haven, but they sometimes killed livestock.',

            paragraph_Where_Did_The_Cheetah_Go_02:
              'Farmers saw cheetahs more frequently and thought their numbers had increased. ' +
              'Cheetahs took the blame for most predator-related livestock losses. ' +
              'Farmers killed cheetahs by the thousands as pests or to seell their skins ' +
              'to the fur trade.',

            paragraph_Where_Did_The_Cheetah_Go_03:
              'By 1975, researches realized that the cheetah was in troubles. CITES ' +
              '(Convention on International Trade of Endangered Species) placed the cheetah ' +
              'on Appendix I, making international trade in live cheetah or cheetah products illegal. ' +
              'Local laws supported CITES in many countries where cheetahs still lived. ' +
              'Researchers began looking for ways to encourage the growth of cheetah populations ' +
              'through land management practices.'
          }
        }
      }
    }
  }
};
