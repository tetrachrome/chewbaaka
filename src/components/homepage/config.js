/**
 * homepage/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 28, 2020
 * Updated  : Aug 04, 2020
 */

export const config = {
  components: {
    landingPageGateway: {
      constants: {
        landingPageGatewayTitle: "Learn about cheetahs",
        landingPageGatewaySubtitle: "Explore the Cheetah Museum at Cheetah Conservation Fund"
      },
      dynamicStyleGroups: {
        centerSegmentStyleClassNames: [
          "LandingPageGatewayBackgroundStyle1",
          "LandingPageGatewayBackgroundStyle2",
          "LandingPageGatewayBackgroundStyle3",
          "LandingPageGatewayBackgroundStyle4",
        ]
      }
    }
  }
}
