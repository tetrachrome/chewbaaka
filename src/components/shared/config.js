/**
 * shared/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 11, 2020
 * Updated  : Jul 11, 2020
 */

export const config = {
  imageCollage: {
    images:  [
      "https://th.thgim.com/sci-tech/energy-and-environment/yrktd/article30295376.ece/ALTERNATES/FREE_960/15SMafrican-cubsjpgs",
      "https://ichef.bbci.co.uk/news/320/cpsprodpb/27A3/production/_96974101_gettyimages-cheetah.jpg",
      "https://idsb.tmgrup.com.tr/ly/uploads/images/2020/05/20/thumbs/800x531/36753.jpg",
      "https://travelafricamag.com/wp-content/uploads/2018/01/HR_Mike-Dexter_20171107-IMG_5091.jpg",
      "https://media.mehrnews.com/d/2019/09/01/4/3224140.jpg",
      "https://images.theconversation.com/files/152362/original/image-20170111-6447-606129.jpg?ixlib=rb-1.1.0&q=45&auto=format&w=926&fit=clip",
      "https://www.treehugger.com/thmb/ahEMfgEkro4OBW8hOdenpM3V2bw=/768x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/cheetah-licking-cub-4af516e956004c01a97b4a51175fd8ff.jpg"
    ],
    collageCellContent: [
      {
        title: "Get Involved",
        subtitle: "Every action counts",
        href: "https://cheetah.org/get-involved/ways-to-give/"
      },
      {
        title: "International Cheetah Day",
        subtitle: "Cheetahs deserve a holiday of their own",
        href: "https://internationalcheetahday.com/"
      },
      {
        title: "Cheetah on IUCN",
        subtitle: "Latest status on the species",
        href: "https://www.iucnredlist.org/es/species/219/13034574"
      },
      {
        title: "Join Forces",
        subtitle: "Work with other organizations",
        href: "https://wildnet.org/"
      },
      {
        title: "Take Action",
        subtitle: "Be a global citizen",
        href: "https://www.globalcitizen.org/en/take-action/"
      },
      {
        title: "Wildlife Conservation",
        subtitle: "It's not too late",
        href: "https://www.worldwildlife.org/initiatives/wildlife-conservation"
      },
      {
        title: "Cheetah on National Geographic",
        subtitle: "Watch the great documentary",
        href: "https://www.youtube.com/watch?v=zfyuzX_SAH4"
      },
    ]
  }  
}
