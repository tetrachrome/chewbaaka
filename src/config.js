/**
 * src/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 09, 2020
 * Updated  : Jul 09, 2020
 */

export const config = {
  styling: {
    mode: 'debug',
    debugStyles: {
      // 'DebugColor': true,
      'DebugWireframe': true,
    }
  }
};
