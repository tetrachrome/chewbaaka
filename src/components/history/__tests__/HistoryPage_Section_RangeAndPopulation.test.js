/**
 * HistoryPage_Section_RangeAndPopulation.test.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 10, 2020
 * Updated  : Jul 10, 2020
 */

import React from 'react';

import { render } from '@testing-library/react';

import { config } from '../config'

import renderer from 'react-test-renderer';

import HistoryPageSectionRangeAndPopulation from '../HistoryPage_Section_RangeAndPopulation';

test('renders HistoryPageSectionRangeAndPopulation component', () => {
  render(<HistoryPageSectionRangeAndPopulation config={config}/>);
});

test('HistoryPageSectionRangeAndPopulation component snapshot', () => {
  const tree = renderer
    .create(
      <HistoryPageSectionRangeAndPopulation config={config}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
