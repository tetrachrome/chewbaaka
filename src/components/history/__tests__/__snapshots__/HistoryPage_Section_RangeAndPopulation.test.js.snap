// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryPageSectionRangeAndPopulation component snapshot 1`] = `
<div>
  <div
    className="ContentPageSectionTemplateOuterContainer ContentPageSectionTemplateOuterContainer_DebugColor DebugWireframe"
  >
    <div>
      <div
        className="ContentPageSubsectionTemplateOuterContainer ContentPageSubsectionTemplateOuterContainer_DebugColor DebugWireframe"
      >
        <div
          className="ContentPageSubsectionTemplateInnerContainer ContentPageSubsectionTemplateInnerContainer_DebugColor DebugWireframe"
        >
          <h3
            className="ContentPageSubsectionTitle ContentPageSubsectionTitle_DebugColor DebugWireframe"
          />
          <div
            className="ContentPageSectionHeadOuterContainer"
          >
            <div
              data-testid="ContentPageSectionAnchorComponentTestId"
              id="range-and-population"
            />
            <div
              className="ContentPageSectionHeadInnerContainer ContentPageSectionHeadInnerContainer_DebugColor DebugWireframe"
            >
              <div
                className="ContentPageSectionHeadTitleContainer ContentPageSectionHeadTitleContainer_DebugColor DebugWireframe"
              >
                <h2
                  className="ContentPageSectionHeadTitle ContentPageSectionHeadTitle_DebugColor DebugWireframe"
                >
                  Range and Population
                </h2>
              </div>
              <div
                className="ContentPageIntroParagraphContentContainer"
              >
                <p
                  className="ContentPageSectionSubtitleText"
                >
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="ContentPageSectionTemplateInnerContainer ContentPageSectionTemplateInnerContainer_DebugColor DebugWireframe"
    >
      <div>
        <div
          className="HistoryPageSubsectionNamibiaOuterContainer"
        >
          <div
            className="ContentPageSubsectionTemplateOuterContainer ContentPageSubsectionTemplateOuterContainer_DebugColor DebugWireframe"
          >
            <div
              className="ContentPageSubsectionTemplateInnerContainer ContentPageSubsectionTemplateInnerContainer_DebugColor DebugWireframe"
            >
              <h3
                className="ContentPageSubsectionTitle ContentPageSubsectionTitle_DebugColor DebugWireframe"
              >
                Namibia - Cheetah Capital of the World
              </h3>
              <div
                className="HistoryPageSubsectionNamibiaInnerContainer"
              >
                <div>
                  <div
                    className="ContentPageSubsectionTwoColumnContentTemplateOuterContainer ContentPageSubsectionTwoColumnContentTemplateOuterContainer_DebugColor DebugWireframe"
                  >
                    <div
                      className="ContentPageSubsectionTwoColumnContentTemplateInnerContainer"
                    >
                      <div>
                        <div
                          className="ui vertically divided grid"
                        >
                          <div
                            className="two column row"
                          >
                            <div
                              className="column"
                            >
                              <div
                                className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer_DebugColor DebugWireframe"
                              >
                                <div>
                                  <p>
                                    Namibia has the world's largest cheetah population. Approximately 3000 cheetahs share the land with humans, livestock, and wildlife. Today, the status of the Namibian cheetah is stabilizing. During the 1980s, the population of Namibina cheetahs declined by half. In the 10-year period, nearly 7,000 cheetahs were removed from the wild. In most countries where cheetahs live, their numbers have been reduced to critical levels. The cheetah's survival worldwide is in human hands.
                                  </p>
                                  <div
                                    className="DisplayInlineBlock"
                                  >
                                    <div
                                      className="ui inverted segment"
                                    >
                                      <div
                                        className="ui orange huge inverted statistic"
                                      >
                                        <div
                                          className="value"
                                        >
                                          21%
                                        </div>
                                        <div
                                          className="label"
                                        >
                                          of the world's wild cheetahs live in Namibia.
                                        </div>
                                      </div>
                                    </div>
                                  </div>
                                  <div
                                    className="DisplayInlineBlock"
                                  >
                                    <div
                                      className="ui inverted segment"
                                    >
                                      <div
                                        className="ui yellow huge inverted statistic"
                                      >
                                        <div
                                          className="value"
                                        >
                                          90%
                                        </div>
                                        <div
                                          className="label"
                                        >
                                          of Namibia's cheetahs live on communal and commercial livestock and game farms.
                                        </div>
                                      </div>
                                    </div>
                                  </div>
                                </div>
                              </div>
                            </div>
                            <div
                              className="column"
                            >
                              <div
                                className="ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer ContentPageSubsectionTwoColumnContentTemplateColumnInnerContainer_DebugColor DebugWireframe"
                              >
                                <div>
                                  <div
                                    className="ImageViewOuterContainer"
                                  >
                                    <div
                                      className="ImageViewInnerContainer"
                                    >
                                      <div
                                        className="ImageViewCore"
                                      >
                                        <img
                                          alt="Cheetah ranges in Namibia (Cheetahs (Acinonyx jubatus) running the gauntlet: an evaluation of translocations into free-range environments in Namibia)"
                                          className="ImageViewImg"
                                          src="https://dfzljdn9uc3pi.cloudfront.net/2015/1346/1/Suppl_Inf_5_cheetah_occurrence_and_suitable_PA.png"
                                          style={
                                            Object {
                                              "height": 640,
                                              "width": 640,
                                            }
                                          }
                                        />
                                        <div
                                          data-testid="ImageViewComponentCaptionPart"
                                        >
                                          <div
                                            className="CaptionContainer"
                                            data-testid="CaptionComponentTestId"
                                          >
                                            <div
                                              className="IconContainer"
                                            >
                                              <i
                                                aria-hidden="true"
                                                className="image icon"
                                                onClick={[Function]}
                                              />
                                            </div>
                                            <p
                                              className="CaptionText"
                                            >
                                              Cheetah ranges in Namibia (Cheetahs (Acinonyx jubatus) running the gauntlet: an evaluation of translocations into free-range environments in Namibia)
                                            </p>
                                          </div>
                                        </div>
                                      </div>
                                    </div>
                                  </div>
                                  <div>
                                    <a
                                      href="https://peerj.com/articles/1346/"
                                    >
                                      <button
                                        className="ui blue icon button"
                                        onClick={[Function]}
                                      >
                                        Cheetahs (Acinonyx jubatus) running the gauntlet: an evaluation of translocations into free-range environments in Namibia
                                        <i
                                          aria-hidden="true"
                                          className="file alternate outline icon"
                                          onClick={[Function]}
                                        />
                                      </button>
                                    </a>
                                  </div>
                                </div>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="HistoryPageSubsectionRoadToExtinctionOuterContainer"
        >
          <div
            className="ContentPageSubsectionTemplateOuterContainer ContentPageSubsectionTemplateOuterContainer_DebugColor DebugWireframe"
          >
            <div
              className="ContentPageSubsectionTemplateInnerContainer ContentPageSubsectionTemplateInnerContainer_DebugColor DebugWireframe"
            >
              <h3
                className="ContentPageSubsectionTitle ContentPageSubsectionTitle_DebugColor DebugWireframe"
              >
                The Road to Extinction ?
              </h3>
              <div
                className="HistoryPageSubsectionRoadToExtinctionInnerContainer"
              >
                <div
                  className="DisplayInlineBlock"
                >
                  <div
                    className="ui inverted segment"
                  >
                    <div
                      className="ui orange huge inverted statistic"
                    >
                      <div
                        className="value"
                      >
                        &lt;7,500
                      </div>
                      <div
                        className="label"
                      >
                        Cheetahs were found in 31 populations in 23 African countries.
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  className="DisplayInlineBlock"
                >
                  <div
                    className="ui inverted segment"
                  >
                    <div
                      className="ui red huge inverted statistic"
                    >
                      <div
                        className="value"
                      >
                        &lt;50
                      </div>
                      <div
                        className="label"
                      >
                        Asian cheetahs left in Iran.
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  className="ContentPageSubsectionThreeColumnContentTemplateOuterContainer ContentPageSubsectionThreeColumnContentTemplateOuterContainer_DebugColor DebugWireframe"
                >
                  <div
                    className="ContentPageSubsectionThreeColumnContentTemplateInnerContainer ContentPageSubsectionThreeColumnContentTemplateInnerContainer_DebugColor DebugWireframe"
                  >
                    <div
                      className="ContentPageSubsectionThreeColumnContentTemplateColumnContainer"
                    >
                      <div
                        className="ContentPageSubsectionThreeColumnContentTemplateColumn ContentPageSubsectionThreeColumnContentTemplateColumn_DebugColor DebugWireframe ContentPageSubsectionThreeColumnContentTemplateLeftColumnContainer ContentPageSubsectionThreeColumnContentTemplateLeftColumnContainer_DebugColor DebugWireframe"
                      >
                        <h2
                          className="ContentPageSubsectionColumnContentTitle ContentPageSubsectionColumnContentTitle_DebugColor DebugWireframe"
                        >
                          The Road to Extinction?
                        </h2>
                        <div>
                          <div>
                            <div>
                              <div>
                                <h3
                                  className="ContentPageSubsectionColumnContentSubtitle ContentPageSubsectionColumnContentSubtitle_DebugColor DebugWireframe"
                                >
                                  1990
                                </h3>
                                <p>
                                  World development, industrialization, automobiles, and aeroplanes...
                                </p>
                                <p>
                                  Through the 1900s, man's inventions seemed limitless, yet there is nothing man-made that rivals the speed and efficiency of the cheetah.
                                </p>
                                <p>
                                  Scientists classified the cheetah as Acinonyx jubatus, yet nobles still referred to it as the "hunting leopard". By the end of the 1800s, cheetahswere a rarity in Asia Minor and Arabia because of their use in the sport of "coursing".
                                </p>
                                <p>
                                  Although it appeared that the cheeatahs had a large range, their numbers within that range existed in small pockets. Cheetahs, farmers and their livestock all preferred open grasslands for their habitat. Increasing agricultural development and new settlements played havoc with remaining cheetah populations.
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                      <div
                        className="ContentPageSubsectionThreeColumnContentTemplateColumn ContentPageSubsectionThreeColumnContentTemplateColumn_DebugColor DebugWireframe ContentPageSubsectionThreeColumnContentTemplateMiddleColumnContainer ContentPageSubsectionThreeColumnContentTemplateMiddleColumnContainer_DebugColor DebugWireframe"
                      >
                        <div>
                          TODO: middle
                        </div>
                      </div>
                      <div
                        className="ContentPageSubsectionThreeColumnContentTemplateColumn ContentPageSubsectionThreeColumnContentTemplateColumn_DebugColor DebugWireframe ContentPageSubsectionThreeColumnContentTemplateRightColumnContainer ContentPageSubsectionThreeColumnContentTemplateRightColumnContainer_DebugColor DebugWireframe"
                      >
                        <h2
                          className="ContentPageSubsectionColumnContentTitle ContentPageSubsectionColumnContentTitle_DebugColor DebugWireframe"
                        >
                          Where Did the Cheetah Go?
                        </h2>
                        <div>
                          <div>
                            <div>
                              <div>
                                <h3
                                  className="ContentPageSubsectionColumnContentSubtitle ContentPageSubsectionColumnContentSubtitle_DebugColor DebugWireframe"
                                >
                                  1975
                                </h3>
                                <p>
                                  As human population grew, the amount of land devoted to livestock farming steadily increased. Livestock filled the open land where cheetahs roamed. Natural prey became scarce. Farmers kill other large predators. Although game reserves protected them, cheetahs could not compete against hyoenas and lions. Farmlands offered cheetahs a safe haven, but they sometimes killed livestock.
                                </p>
                                <p>
                                  Farmers saw cheetahs more frequently and thought their numbers had increased. Cheetahs took the blame for most predator-related livestock losses. Farmers killed cheetahs by the thousands as pests or to seell their skins to the fur trade.
                                </p>
                                <p>
                                  By 1975, researches realized that the cheetah was in troubles. CITES (Convention on International Trade of Endangered Species) placed the cheetah on Appendix I, making international trade in live cheetah or cheetah products illegal. Local laws supported CITES in many countries where cheetahs still lived. Researchers began looking for ways to encourage the growth of cheetah populations through land management practices.
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
